understanding/21/non-text-contrast.html

```diff
@@ -118,10 +118,9 @@ <h4 id="related-focus">Relationship with Focus Visible</h4>
 				<figure id="figure-focus-outer-green">
 					<img src="img/ntc-focus-outer-green.png" alt="Three blue buttons on a white background, the middle has a dark green outline outside of the botton's non-focused boundary." width="400" />
 					<figcaption>
-						The external green indicator (#008000) does contrast with the white background (#FFF) which the component is on, <strong>passing</strong> the criterion. It does not need to contrast with both the component background and the component.
+						The external green indicator (#008000) does contrast with the white background (#FFF) which the component is on, <strong>passing</strong> the criterion. It does not need to contrast with both the component background and the component, as visually the effect is that the button is noticeably larger, and it's not necessary for a user to be able to discern this extra border in isolation. Although this passes non-text contrast, it is not a good indicator unless it is very thick. <span class="wcag22">There is a criterion in WCAG 2.2 that addresses this aspect, <a href="../22/focus-appearance-minimum.html">Focus Appearance</a>
```

Comment:
Since [2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance) ended up a AAA, should that be reflecting in this paragraph?

> There is a <ins>criterion</ins> in WCAG 2.2 that addresses this aspect...